### PR TITLE
[CWS] add feature flag for policy monitoring

### PR DESCRIPTION
**Pull request by safchain**
### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: pkg/security/config/config.go [hunk 1]
```diff
@@ -39,6 +39,8 @@ type Config struct {
 	PoliciesDir string
 	// WatchPoliciesDir activate policy dir inotify
 	WatchPoliciesDir bool
+	// MonitorRules enable rule loaded monitoring
+	PolicyMonitorEnabled bool
```
Comment (YoannGh) [inline pkg/security/config/config.go:43]: nit: comment should be PolicyMonitorEnabled

File: pkg/security/module/module.go [hunk 2]
```diff
@@ -112,7 +112,9 @@ func (m *Module) Init() error {
 	m.apiServer.Start(m.ctx)
 
 	// monitor policies
-	m.policyMonitor.Start(m.ctx)
+	if m.config.PolicyMonitorEnabled {
+		m.policyMonitor.Start(m.ctx)
+	}
```
Comment (sgnn7) [inline pkg/security/module/module.go:117]: If this is a change in default user-visible behavior, we may want to add a changelog entry.

**Review by sgnn7 — APPROVED** [review]
@safchain Left a comment but LGTM

File: pkg/security/module/module.go (hunk 2 above)
Comment (safchain) [reply pkg/security/module/module.go:117]: This monitoring was added during this release and it not a user facing feature yet.

**Review by spikat — APPROVED** [review]

**Pull request merged**